controllers/depresolver/depresolver_validator.go

```diff
@@ -0,0 +1,159 @@
+package depresolver
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+const (
+	//hostNameRegex allows cloud region formats; e.g. af-south-1
+	geoTagRegex = "^[a-zA-Z\\-\\d]*$"
```

Comment:
Are you sure we want to force it ? Also, how did you test the regex, didn't succeed to match region format for me, but I might be missing something

Reply:
@ytsarev good point. Here is regex test:
[expected](https://github.com/AbsaOSS/k8gb/blob/170-include-envvars-into-depresolver/controllers/depresolver/depresolver_test.go#L228)
[corner case](https://github.com/AbsaOSS/k8gb/blob/170-include-envvars-into-depresolver/controllers/depresolver/depresolver_test.go#L217)

Reply:
perfect